destinations/sift/handler.js

```diff
@@ -0,0 +1,86 @@
+// Segment & Sift 
+const SEGMENT_WRITE_KEY = ""
+const endpoint = "https://api.sift.com/v205/events?return_score=true"
+
+async function addScores(fields, res) {
+	var body = {
+		"userId": fields.$user_id,
+		"traits": {
+			"contentAbuseScore": res.score_response.scores.content_abuse.score,
```

Comment:
should we have some checks on this line and the next to make sure the objects exist? (perhaps use lodash's get?  https://lodash.com/docs/#get)